packages/web-components/@groupby/elements-sayt/package.json

```diff
@@ -22,13 +22,18 @@
     "url": "https://github.com/groupby/elements-view.git",
     "directory": "packages/web-components/@groupby/elements-sayt"
   },
-  "author": "",
+  "author": "Groupby Inc",
   "license": "MIT",
   "devDependencies": {
     "html-entities": "^1.2.1"
```

Comment:
This should be moved to the `devDependencies` at the top level. Confirm with other packages as well, please.